tests/integration/components/description-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'website-www/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | description', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`<Description />`);
+
+    assert.dom(this.element).hasText('');
```

Comment:
Please add few more assertions to test out actual description content.

Reply:
Tests will be done in separate PRs. I have mentioned the issue ticket for the same here https://github.com/Real-Dev-Squad/website-www/pull/330#issue-1469904836

Reply:
Ok Got it. Thank you.